includes/Core/Authentication/Clients/Google_Proxy_Client.php

```diff
@@ -227,4 +228,19 @@ private function fetchAuthToken( OAuth2 $auth, callable $http_handler = null ) {
 
 		return $credentials;
 	}
+
+	/**
+	 * Gets the base URL to the authentication service.
+	 *
+	 * @since n.e.x.t
+	 * @static
+	 *
+	 * @return string Base URL to the authentication service, without trailing slash.
+	 */
+	public static function get_base_url() {
```

Comment:
Let's move this as non-static, as well as `OAuth_Client::get_proxy_setup_url()` and `OAuth_Client::get_proxy_permissions_url()` to `Google_Proxy`.